index.bs

```diff
@@ -3491,15 +3512,40 @@ and at least one of the following three conditions is true:
             and it is not the case that both are [=callback interfaces=].
         1.  The interface type is not a [=callback interface=].
     </ol>
-1.  One type is a [=union type=] or nullable union type,
-    the other is neither a union type nor a nullable union type, and each
-    [=member type=] of the first is distinguishable
-    with the second.
-1.  Both types are either a union type or nullable union type, and each member type of the one
-    is distinguishable with each member type of the other.
-
-Note: [=Promise types=] do not appear in the above table, and as a consequence
-are not distinguishable with any other type.
+
+    <div class="example" id="example-distinguishability">
+        {{double}} and {{DOMString}} are distinguishable because
+        there is a ● at the intersection of [=numeric types=] with [=string types=].
+
+        {{double}} and {{long}} are not distinguishable because they are both [=numeric types=],
+        and there is no ● or letter at the intersection of [=numeric types=] with [=numeric types=].
+
+        Given:
+
+        <pre highlight="idl">
+            callback interface Foo {
+                attribute DOMString attr1;
+            };
+
+            interface Bar {
+                attribute DOMString attr2;
+            };
+
+            dictionary Baz {
+                DOMString field1;
+            };
+        </pre>
+
+        <code>Foo</code> is distinguishable from <code>Bar</code>
+        because the pair satisfies note (a),
+        but it is not distinguishable from <code>Baz</code>
+        because that pair does not satisfy note (b).
+
+        [=Promise types=] do not appear in the above table, and as a consequence are
```

Comment:
I moved it because it's an example of how to interpret the table in order to find whether some types are distinguishable. That said, there's not much difference between a note and an example, so I could move it back.